balboa-service-jms/src/main/java/com/socrata/balboa/jms/activemq/Consumer.java

```diff
@@ -37,28 +39,34 @@ public void onMessage(Message payload) {
         long start = System.currentTimeMillis();
         String messageText = null;
         try {
-            TextMessage text = (TextMessage) payload;
-            messageText = text.getText();
+            // Before we even consider chunking a large message, check if we even need to write to send out the message
+            if (stopWrites) {
+                session.commit();
+                log.debug("Config was set to stop writes from Balboa to Cassandra. Ignoring message");
```

Comment:
This was the only thing added to this file. The git diff makes this look ugly tho